src/components/layouts/Navigation.tsx

```diff
@@ -4,16 +4,114 @@ import Link from "next/link";
 export const Navigation = () => {
   return (
     <Header>
-      <Link href="/">down to earth.</Link>
       <Navigator>
-        <Link href="/about">about</Link>
-        <Link href="/products">products</Link>
-        <Link href="/events">events</Link>
+        <Link href="/"><img src="/assets/logo.png" alt="logo" /></Link> 
+        <ul>
+          <li><Link href="/about">about</Link></li>
+          <li><Link href="/products">products</Link></li>
+          <li><Link href="/events">events</Link></li>
+        </ul> 
+        <label htmlFor="search-bar"></label>
+        <input type="text" placeholder="검색어를 입력해주세요"/> 
+        <LoggedInContainer>
+          <img src="/assets/user.svg" alt="user icon" /> 
+          <CartIconContainer>
+            <div>
+              <span>3</span>
+            </div>
+            <img src="/assets/cart.svg" alt="cart icon" />
+          </CartIconContainer> 
+        </LoggedInContainer>
+        <JoinContainer>
+          <Link href="/login">로그인</Link>
+          <Link href="/join">회원가입</Link>
+        </JoinContainer>
       </Navigator>
     </Header>
   );
 };
 
-const Header = styled.header``;
+const Header = styled.header`
+  display: flex; 
+  justify-content: space-between; 
+  align-items: center;
+  width: 100%; 
+  height: 80px;
+  box-sizing: border-box; 
+  padding: 0 100px; 
+`
 
-const Navigator = styled.nav``;
+const Navigator = styled.nav`
+  width: 100%; 
+  display: flex; 
+  justify-content: space-between;
+  align-items: center; 
+
+  img {
+    height: 25px;
+    margin-bottom: 10px;
+    cursor: pointer; 
+  }
+
+  ul { 
+    display: flex; 
+    justify-content: space-evenly;
+    font-size: 1.8rem; 
+    font-weight: bold; 
+    width: 30rem; 
+    flex-shrink: 0;
+    
+    li:hover {
+      opacity: 0.3; 
+    }  
+  }
+
+  input {
+    background: url("/assets/search.svg") no-repeat 1rem 0.9rem; 
+    width: 250px;
+    height: 40px; 
+    border-radius: 40px; 
+    border: 1px solid #A1A1A1; 
+    box-sizing: border-box;
+    padding:  0 40px;
+  } 
+`
+
+// 로그인 성공 시 사용
+const LoggedInContainer = styled.div`
+  width: 10rem; 
+  display: flex; 
+  align-items: center;
+  justify-content: space-between; 
+  margin-top: 10px; 
+  display: none;
+  
+`
+const JoinContainer = styled.div`
+  width: 120px;
+  display: flex; 
+  justify-content: space-between;
+  flex-shrink: 0; 
+  font-size: 1.4rem;
+`
+
+const CartIconContainer = styled.div`    
+  position: relative; 
+
+  div {  
+    position: absolute; 
+    left: 13px; 
+    top: 15px;
+    width: 20px; 
+    height: 20px;
+    background: #847258;
```

Comment:
p3: mainColor는 theme에 등록해놨고 주된 색상은 thme에 등록하셔서 사용하시면 좋을 것 같습니다.
```suggestion
    background: thme.mainColor
    
    예시 :
    const Text = styled.p(props => ({
    color: props.theme.mainColor,
}));
```
참조자료 링크: https://tech.devsisters.com/posts/react-extend-theme/
사용관련: https://stackoverflow.com/questions/69718472/not-able-to-access-theme-in-styled-component-using-emotion-styled